src/DssSpell.t.sol

```diff
@@ -36,15 +36,20 @@ interface BridgeLike {
     function l2TeleportGateway() external view returns (address);
 }
 
-interface LineMomLike {
-    function owner() external view returns (address);
-    function authority() external view returns (address);
-    function autoLine() external view returns (address);
-    function vat() external view returns (address);
-    function ilks(bytes32) external view returns (uint256);
-    function wipe(bytes32 ilk) external returns (uint256);
+// For PE-1208
+interface RwaUrnLike {
+    function hope(address) external;
+    function draw(uint256) external;
 }
 
+interface RwaOutputConduitLike {
+    function hope(address) external;
+    function bud(address) external view returns (uint256);
+    function pick(address) external;
+    function push() external;
+}
```

Comment:
I think we remove these. It looks like they are in dss-interfaces

Reply:
I haven't imported interfaces on mainnet spell but in line with other removals above I will remove this for Goerli